video_xblock/static/js/player_state.js

```diff
@@ -29,18 +29,27 @@ var player_state = {
 var xblockUsageId = window.location.hash.slice(1);
 
 /** Restore default or previously saved player state */
-var setInitialState = function(player, state) {
-  if (state.currentTime > 0) {
-    player.currentTime(state.currentTime);
-  }
-  player
-    .volume(state.volume)
-    .muted(state.muted)
-    .playbackRate(state.playbackRate);
+var setInitialState = function (player, state) {
+    var stateCurrentTime = state.currentTime;
+    var videoXblockState = localStorage.getItem('videoXblockState');
+    if (videoXblockState){
+        videoXblockState=JSON.parse(videoXblockState);
+        if (videoXblockState['{{ video_player_id }}'] && videoXblockState[
```

Comment:
Either move last `videoXblockState[` to next line or make this clause a single line.